### ISSUE TITLE
Consider eslint-plugin-jsx-a11y

### ISSUE THREAD

**Issue by whymarrh** [MetaMask/eslint-config#42]
Refs #10

A React config should have a11y rules as well: [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)
